src/index.js

```diff
@@ -18,15 +18,42 @@ import Player from './Player';
 import Round from './Round';
 import Turn from './Turn';
 
+function createPlayers(Player1, Player2, Player3) {
+  let player1 = new Player(Player1, 1)
+  let player2 = new Player(Player2, 2)
+  let player3 = new Player(Player3, 3)
+  return [player1, player2, player3]
+}
+
+// $(window).load(function() {
+//   // will first fade out the loading animation
+// $('.welcome-message').delay(1500).fadeOut("slow");
+//   // will fade out the whole DIV that covers the website.
+// //$("#preloader").delay(2000).fadeOut("slow");
+// })
 $(document).ready(function() {
-
-  $('#player1-name__submit').click( () => {
- getPlayerName('#player1-name__input', '#player1-name__display')
+  $('.user-name-inputs').hide();
+  $('.puzzle-table').hide();
+  $('.welcome-message').delay(2500).fadeOut("slow");
+  $('.user-name-inputs').append(`
+  <input type="text"id="player1-name__input"> Player 1 </input>
+  <p></p>
+  <input type="text"id="player2-name__input"> Player 2 </input>
+  <p></p>
+  <input type="text"id="player3-name__input" label="Player3">Player 3:</input>
+  <p></p>
+  <button id="player3-name__submit">Game on!</button>`).delay(3000).fadeIn('slow');
+  //$('.user-name-inputs').delay(3000).fadeIn("slow");
+
+  $('#player3-name__submit').click( () => {
+ getPlayerName('#player3-name__input', '#player3-name__display');
+ $('.user-name-inputs').fadeOut();
```

Comment:
The DTR is that we are querySelecting with jQuery by #id.

Reply:
Identifier changed to ID